tests/catalog.bom

```diff
@@ -6,11 +6,21 @@ brooklyn.catalog:
   itemType: template
   iconUrl: https://pbs.twimg.com/profile_images/770492216863236096/8igZkijg.jpg
   item:
+    brooklyn.parameters:
+    - name: amp.download.url
+      label: AMP URL
+      description: |
+        URL of the AMP to use
+      type: string
+      default: http://downloads.cloudsoftcorp.com/amp/latest/cloudsoft-amp-karaf-latest-noarch.rpm
```

Comment:
not sure I like the default in the test. It can lead to a situation where there is a bug setting this and we don't notice